web_timeline/static/src/js/web_timeline.js

```diff
@@ -278,7 +284,7 @@ odoo.define('web_timeline.TimelineView', function (require) {
             }
             self.last_group_bys = n_group_bys;
             // gather the fields to get
-            var fields = _.compact(_.map(["date_start", "date_delay", "date_stop", "progress"], function (key) {
+            var fields = _.compact(_.map(["date_start", "date_delay", "date_stop", "progress", "tooltip"], function (key) {
```

Comment:
AFAIK, this makes `tooltip` as a mandatory field, which is not the idea.

Reply:
It does not work without that line. The attr "tooltip" is not loaded. Is there another place to to this in the code ?

Reply:
You have to load conditionally the value if present